Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/search/Searcher.java

```diff
@@ -156,26 +146,22 @@ public JSONObject findDocument(String query) throws DataException {
      *            as String
      * @return search result
      */
-    public JSONObject findDocument(String query, String sort) throws DataException {
+    public JsonObject findDocument(String query, String sort) throws DataException {
         SearchRestClient restClient = initiateRestClient();
-        JSONParser parser = new JSONParser();
 
         String response = restClient.getDocument(query, sort, 0, 1);
-        try {
-            JSONObject jsonObject = (JSONObject) parser.parse(response);
-            if (jsonObject.containsKey("hits")) {
-                JSONObject hits = (JSONObject) jsonObject.get("hits");
-                JSONArray inHits = (JSONArray) hits.get("hits");
-                if (!inHits.isEmpty()) {
-                    return (JSONObject) inHits.get(0);
-                }
-            } else {
-                return jsonObject;
+        JsonReader jsonReader = Json.createReader(new StringReader(response));
```

Comment:
Same here. `JsonReader` is closeable so using a try-with-resources statement for closing the object is maybe a good idea.